src/vscode-bicep/package.json

```diff
@@ -347,7 +347,7 @@
             "title": "Open a Bicep File",
             "description": "To get started with Bicep extension features, open or create a Bicep file (any file with '.bicep' extension).\n[Open Bicep file](command:bicep.gettingStarted.openBicepFile)\n[Create Bicep file](command:bicep.gettingStarted.createBicepFile)\nTip: [Close the side bar for more space](command:workbench.action.closeSidebar)",
             "media": {
-              "image": "media/walkthroughs/gettingStarted/1_BicepLogoImage.png",
+              "image": "media/walkthroughs/gettingStarted/0_bicepLogoImage.png",
```

Comment:
replace all error

Reply:
Ahhh thank you!

Updated here:
https://github.com/Azure/bicep/commit/169985ad43c52e726172b13e4e4f1e014e1d552f